src/main/java/com/sendgrid/helpers/mail/Mail.java

```diff
@@ -292,6 +295,38 @@ public void addAttachments(Attachments attachments) {
     this.attachments = addToList(newAttachment, this.attachments);
   }
 
+  /**
+   * Downloads file from Dropbox based on given URL and adds an {@link Attachments} of that file to
+   * {@link #attachments}.
+   * @param dropboxURL The Dropbox URL that is to be attached.
+   */
+  public void addDropboxAttachment(String dropboxURL) {
```

Comment:
Prefer to create a new class that extends `Attachment` (e.g., `DropboxAttachment`) that you can construct and add to the mail object. This allows the logic to be isolated from the rest of the mail logic.